std/typecons.d

```diff
@@ -2185,6 +2185,69 @@ Params:
         _isNull = false;
     }
 
+    /**
+      If they are both null, then they are equal. If one is null and the other
+      is not, then they are not equal. If they are both non-null, then they are
+      equal if their values are equal.
+      */
+    bool opEquals()(auto ref inout(typeof(this)) rhs) inout
+    {
+        if(_isNull)
```

Comment:
The code style is to have a whitespace after `if` (that's why CircleCi is failing)

Btw it's also not covered by the unittests.